### DISCUSSION TITLE
Bridge help chat

### DISCUSSION THREAD

**Discussion by toastal** [argyleink/open-props#434] [Ideas]
In my opinion, it’s unreasonable for open source projects to have chat happen on proprietary platforms that require an account & agreeing to American company ToS to participate. This chat also becomes a black hole for support as search engines can’t crawl this text leading answers to be lost behind a walled garden & causing already-answered questions to have lost their reference.

At a minimum I would expect ideally a decentralized mirror on an open network such as XMPP or Matrix or a centralized but open network like IRC so users who don’t feel comfortable using or have regional blockage/hardware/accessibility/privacy reason(s) they can’t use Discord or its client can still participate.

**Comment by argyleink** [comment]
this is a fine place to ask questions and get help as well. no one is forced into discord

**Reply by toastal** [reply]
The first subheading in the README says help is on Discord. The callout on the website also only states “Need help? Ask a question on Discord”.

But also while Microsoft GitHub’s Discussions don’t have the SEO blackhole problem, it still requires an account & agreement to the [lengthy ToS](https://docs.github.com/en/site-policy/github-terms/github-terms-of-service) with Microsoft on a proprietary platform. Being a US-based platform there are issues with [censorship](https://en.wikipedia.org/wiki/Censorship_of_GitHub) & [banning under sanctions](https://techcrunch.com/2019/07/29/github-ban-sanctioned-countries/).
